ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/JsonRpcErrorHandler.java

```diff
@@ -47,11 +47,12 @@ public void handle(final RoutingContext context) {
     final int statusCode =
         context.statusCode() == -1 ? INTERNAL_SERVER_ERROR.code() : context.statusCode();
     LOG.debug(
-        "Failed to correctly handle request. method: {}, uri: {}, body: {}, Error body: {}",
+        "Failed to correctly handle request. method: {}, uri: {}, body: {}, Error body: {}, Failure: {}",
         context.request()::method,
         context.request()::absoluteURI,
         () -> jsonRpcRequest.map(Json::encodePrettily).orElse(context.getBodyAsString()),
-        () -> Json.encode(errorResponse));
+        () -> Json.encode(errorResponse),
+        () -> Json.encode(context.failure()));
```

Comment:
We don't want to json encode the the exception, it should be just returned as is so the stack trace shows in the log

Reply:
You can't mix getting stack traces with using `{}` placeholders.

Reply:
fixed